package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "construct-style-sheets-polyfill",
   "version": "2.1.0",
-  "description": "Constructible style sheets/ adopted style sheets polyfill",
+  "description": "Constructible style sheets/adopted style sheets polyfill",
   "main": "./adoptedStyleSheets.js",
-  "module": "./adoptedStyleSheets.min.js",
+  "module": "./adoptedStyleSheets.js",
```

Comment:
Well, the problem here is that almost all the most popular tools (e.g., webpack, rollup, pikapkg) use the path from `module` field of the `package.json` as a path to the library main file. And if there is a minified file, it will be used as is. 

It produces a lot of disadvantages. E.g., when I faced some bugs in the polyfill, I wasn't able even to debug it in development mode because of minification. I had to copy the non-minified sources to the `adoptedStyleSheets.min.js` file, restart my webpack build to catch new changes and then stay care of updating sources because each update replaces this file to the minified version. And all this work is while the minification in the consuming project can be done with one command in the toolset. 

That's the reason I added this change to the PR. I also support Henry Zhu's thoughts expressed in the [post](https://babeljs.io/blog/2018/06/26/on-consuming-and-publishing-es2015+-packages) in Babel blog.